docs/pages/quickstart.mdx

```diff
@@ -61,13 +61,31 @@ To install Admyral, simply install the Admyral pip package using your favorite d
 
 </Tabs>
 
+<details>
+    <summary> Optionally, specify custom ports for the needed services </summary>
+    
+    Copy the content of `.env.example` file to a new file called `.env`:    
+    ```bash
+    cp .env.example .env
+    ```
+
+    Open the `.env` file and adjust the ports as needed.
+
+    Source the `.env` file to load the environment variables:
+
+    ```bash
+    source .env
+    ```
+
+</details>
+
 To launch Admyral use the following command:
 
 ```bash
 admyral up
 ```
 
-You can open the Admyral UI by visiting `http://localhost:3000` in your browser.
+You can open the Admyral UI by visiting `http://localhost:3000` (if no custom port was specified for `ADMYRAL_WEB_PORT`) in your browser.
```

Comment:
I would prefer the following formulation a bit more:

```suggestion
You can open the Admyral UI by visiting http://localhost:3000 in your browser. If a custom port was defined using `ADMYRAL_WEB_PORT`, use the defined port number instead of `3000`.
```

I would then also use a similar one for the other changes.